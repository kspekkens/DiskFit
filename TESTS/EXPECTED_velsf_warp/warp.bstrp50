EXAMPLE/VELS/FITS/velsf_warp.inp                                                                    
    lpa:  T    leps: T lcentre: T lsystemi T lradial: F lnax:    F luncert: T
#nunc   npars    ntot   nellip  nminr  nmaxr seed
      5      7     26     25      1     25    -50
 
 Bootstrap number:            0
   2.3561944961547852       0.44999998807907104        130.50000000000000        126.50000000000000        90.000000000000000        0.0000000000000000        0.0000000000000000     
   62.8284531       117.649475       131.707275       143.440170       144.796143       150.194702       152.619995       154.380524       155.177368       161.963760       163.843811       173.299850       178.320221       183.559586       190.035828       189.495758       191.297485       192.739639       192.225815       193.348740       192.739227       193.850815       192.876434       194.095184       201.226196       100.006088    

 Bootstrap number:     1/    5  Chisq     17.48715
   2.1310424975744753       0.49916437100749245        128.53478846250013        128.70349171019279        62.257369502000216       -5.2092625737112382E-002  0.25846563715048393     
   59.6524353       115.440910       128.354324       142.862244       144.231750       154.335938       158.159409       159.596191       158.720825       164.859314       169.889297       180.139297       187.540604       192.658752       197.722626       196.718887       200.988617       197.959579       195.492844       194.810944       190.545074       192.490875       193.554657       191.638489       203.423676       100.159988    

 Bootstrap number:     2/    5  Chisq     16.04028
   2.1961649450922764       0.49758132922365034        128.64249946430675        128.30174320631252        68.448275276860429        6.3655132879771488E-003  0.12666582854662534     
   65.0116959       123.878281       137.068787       154.440872       151.629440       165.508469       159.220398       158.360764       158.161652       159.509689       161.090195       167.477570       173.682388       179.582474       185.409607       186.392944       191.756805       194.679367       195.342957       194.875214       194.132202       194.969925       195.258804       191.827026       199.788727       99.6104431    

 Bootstrap number:     3/    5  Chisq     18.58699
   2.1171959876687381       0.54448866317113742        128.55634581323315        128.57162270862645        9.0000016925681834       -1.3542974547910544E-002  0.26862987010252115     
   65.3577271       118.819519       128.023972       147.794556       145.253464       154.814926       152.933456       156.398300       153.303162       160.476990       160.405518       167.514664       175.996231       177.891006       190.215515       189.753326       190.702759       190.570938       190.134506       189.371338       190.098404       190.398117       189.744080       189.818375       199.024994       100.133888    

 Bootstrap number:     4/    5  Chisq     18.48688
   2.2017930655375584       0.47224529706571750        128.45458208332630        128.42231972270423        9.0001891361773243        1.9558952951038766E-002  0.10242758306283252     
   68.1730270       122.344505       134.961777       146.583679       146.117950       150.161957       153.322281       160.289841       163.457413       171.090393       175.723404       186.613831       190.985489       194.170441       196.243607       192.280685       189.814285       188.937744       187.548599       187.568985       188.384033       190.290024       191.026794       194.961533       201.433090       99.9783630    

 Bootstrap number:     5/    5  Chisq     19.75701
   2.1810672631814119       0.51186523966795583        128.11619845512593        128.71592098250932        41.373119682432979       -5.0357453898609308E-002  0.16632878057495976     
   54.5564384       102.860123       120.304283       133.079498       139.403381       144.854263       151.149628       157.460968       158.042374       169.395920       168.368820       177.341064       182.545074       186.318771       191.691452       189.339294       190.328232       188.987762       189.864120       190.357727       192.261795       191.564713       191.440903       187.798508       192.220474       100.282593    

