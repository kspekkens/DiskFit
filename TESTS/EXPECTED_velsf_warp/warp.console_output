 DiskFit (v1.2.2)  Copyright (C) 2015, 2017, Jerry Sellwood and Kristine Spekkens

 This program comes with ABSOLUTELY NO WARRANTY.  It is free software
 and you are welcome to redistribute it under certain conditions.
 See <http://www.gnu.org/licenses/> for details.

 Enter input parameter file name

 Read in input parameters from: EXAMPLE/VELS/FITS/velsf_warp.inp                                                                    

Input disk eps: (1-b/a)  0.45
Input disk PA (N->E; deg) =  45.00
Input x-center (data units) =   130.50
Input y-center (data units) =   126.50
Input disk systemic velocity (km/s) =   110.00

 Fitting for:
 disk PA
 disk ellipticity
 disk x-center, y-center
 disk systemic velocity
 warp model parameters
 --> Other attributes fixed to input values

 Velocity field components extracted at radii:
    3.0    9.0   15.0   21.0   27.0   33.0   39.0   45.0   51.0   57.0
   63.0   69.0   75.0   81.0   87.0   93.0   99.0  105.0  111.0  117.0
  123.0  129.0  135.0  141.0  147.0

 Warp fitting for: inner radius max eps max PA

 Read in measured velocities and uncertainties from:
 EXAMPLE/VELS/FITS/vels.fits


 Number of datapoints read in:        1224
 Number of datapoints to be used in fit:        1224
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   45.0000  0.450000  130.5000  126.5000   90.0000    0.0000    0.0000
starting iteration    1 best reduced chisq so far: 2.636E+02
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   45.0000  0.450000  130.5000  126.5000   90.0000    0.0000    0.0000
starting iteration    2 best reduced chisq so far: 5.200E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   37.9969  0.502700  126.7735  127.7304   92.0004   -0.0138    0.0592
starting iteration    3 best reduced chisq so far: 3.742E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   37.3775  0.503988  127.8515  128.2408   31.2097   -0.0017    0.0628
starting iteration    4 best reduced chisq so far: 3.264E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   36.4657  0.507861  128.2338  128.3999    9.0006   -0.0012    0.0799
starting iteration    5 best reduced chisq so far: 2.981E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   35.6981  0.508328  128.3465  128.4541    9.0000   -0.0008    0.0994
starting iteration    6 best reduced chisq so far: 2.545E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   33.0169  0.506541  128.5651  128.5438    9.0000   -0.0080    0.1719
starting iteration    7 best reduced chisq so far: 2.537E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   33.0054  0.504893  128.5092  128.5393    9.0000   -0.0165    0.1737
starting iteration    8 best reduced chisq so far: 2.524E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   32.9164  0.515554  128.5016  128.5413    9.0000   -0.0389    0.1822
starting iteration    9 best reduced chisq so far: 2.522E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   32.9253  0.517323  128.5047  128.5409    9.0000   -0.0384    0.1815
starting iteration   10 best reduced chisq so far: 2.520E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   32.9289  0.518668  128.5047  128.5424    9.0000   -0.0397    0.1803
starting iteration   11 best reduced chisq so far: 2.519E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   32.9308  0.521425  128.5007  128.5400    9.0000   -0.0419    0.1795
starting iteration   12 best reduced chisq so far: 2.519E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   32.9342  0.521285  128.5020  128.5410    9.0000   -0.0417    0.1791

 Done minimization
 Total number of iterations:           12
 Minimum chi^2 found:    25.1926689    

 Wrote:  EXAMPLE/VELS/FITS/OUT/warp.mod.fits
 Wrote:  EXAMPLE/VELS/FITS/OUT/warp.res.fits

 Determining uncertainties...
 Beginning bootstrap


 Bootstrap number:            1
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   45.0000  0.450000  130.5000  126.5000   90.0000    0.0000    0.0000
starting iteration    1 best reduced chisq so far: 3.306E+02
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   45.0000  0.450000  130.5000  126.5000   90.0000    0.0000    0.0000
starting iteration    2 best reduced chisq so far: 5.613E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   36.6271  0.499370  126.6659  127.8282  106.3293    0.0482    0.0650
starting iteration    3 best reduced chisq so far: 3.747E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   35.8564  0.494594  127.8888  128.4419   81.8970    0.0677    0.0877
starting iteration    4 best reduced chisq so far: 2.510E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   33.1186  0.463111  129.0573  128.8361   50.2009    0.0790    0.1741
starting iteration    5 best reduced chisq so far: 2.121E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   32.0106  0.446702  128.8396  128.7808   35.8061    0.0711    0.1907
starting iteration    6 best reduced chisq so far: 1.993E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   31.2993  0.443328  128.5084  128.6930   32.6077    0.0593    0.2232
starting iteration    7 best reduced chisq so far: 1.962E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   31.3614  0.448512  128.5012  128.6844   36.3591    0.0522    0.2243
starting iteration    8 best reduced chisq so far: 1.789E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   32.0763  0.486615  128.4808  128.7042   62.0916   -0.0189    0.2427
starting iteration    9 best reduced chisq so far: 1.762E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   32.2049  0.490794  128.5215  128.6990   63.3099   -0.0286    0.2525
starting iteration   10 best reduced chisq so far: 1.760E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   32.1697  0.492368  128.5335  128.7014   63.2332   -0.0329    0.2564
starting iteration   11 best reduced chisq so far: 1.756E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   32.0896  0.497041  128.5718  128.7089   61.8963   -0.0428    0.2598
starting iteration   12 best reduced chisq so far: 1.750E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   32.0070  0.497720  128.5684  128.7136   61.5767   -0.0469    0.2594
starting iteration   13 best reduced chisq so far: 1.749E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   32.0692  0.498846  128.5401  128.7055   61.8163   -0.0516    0.2585
starting iteration   14 best reduced chisq so far: 1.749E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   32.0937  0.499050  128.5345  128.7034   62.2300   -0.0519    0.2584

 Done minimization
 Total number of iterations:           14
 Minimum chi^2 found:    17.4871521    



 Bootstrap number:            2
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   45.0000  0.450000  130.5000  126.5000   90.0000    0.0000    0.0000
starting iteration    1 best reduced chisq so far: 2.304E+02
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   45.0000  0.450000  130.5000  126.5000   90.0000    0.0000    0.0000
starting iteration    2 best reduced chisq so far: 3.143E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   38.3082  0.504564  127.2775  127.6529  113.8838   -0.0545    0.1213
starting iteration    3 best reduced chisq so far: 1.794E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   37.2071  0.517721  128.6632  128.3555   91.8003   -0.0388    0.1110
starting iteration    4 best reduced chisq so far: 1.696E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   36.7299  0.517008  128.6632  128.3034   88.7911   -0.0383    0.1281
starting iteration    5 best reduced chisq so far: 1.639E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   36.0120  0.511170  128.6535  128.3083   80.8682   -0.0362    0.1474
starting iteration    6 best reduced chisq so far: 1.628E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   35.9852  0.508744  128.6546  128.3100   78.4216   -0.0243    0.1432
starting iteration    7 best reduced chisq so far: 1.605E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   35.9199  0.498062  128.6477  128.3044   71.2112    0.0062    0.1266
starting iteration    8 best reduced chisq so far: 1.604E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   35.8539  0.497032  128.6444  128.3021   69.4957    0.0072    0.1275
s starting iteration    9 best reduced chisq so far: 1.604E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   35.8343  0.496895  128.6444  128.3007   68.3033    0.0070    0.1267
starting iteration   10 best reduced chisq so far: 1.604E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   35.8301  0.497197  128.6444  128.3020   68.3181    0.0069    0.1266
starting iteration   11 best reduced chisq so far: 1.604E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   35.8308  0.497428  128.6424  128.3008   68.3955    0.0066    0.1266

 Done minimization
 Total number of iterations:           11
 Minimum chi^2 found:    16.0402813    



 Bootstrap number:            3
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   45.0000  0.450000  130.5000  126.5000   90.0000    0.0000    0.0000
starting iteration    1 best reduced chisq so far: 2.583E+02
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   45.0000  0.450000  130.5000  126.5000   90.0000    0.0000    0.0000
starting iteration    2 best reduced chisq so far: 6.453E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   39.5896  0.534084  126.6836  127.7298  106.3287   -0.0066    0.1294
starting iteration    3 best reduced chisq so far: 4.136E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   38.0809  0.551602  127.9298  128.3333   85.9787    0.0077    0.1313
starting iteration    4 best reduced chisq so far: 3.008E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   35.0821  0.563580  128.7716  128.5785   42.3575    0.0276    0.1592
s starting iteration    5 best reduced chisq so far: 2.515E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   34.1695  0.542329  128.6790  128.5673   25.6500    0.0407    0.1677
starting iteration    6 best reduced chisq so far: 2.237E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   33.3780  0.529627  128.6820  128.5720   14.9854    0.0482    0.1876
starting iteration    7 best reduced chisq so far: 2.081E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   32.9234  0.521146  128.6176  128.5638    9.0002    0.0503    0.2023
starting iteration    8 best reduced chisq so far: 2.018E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   32.8122  0.516300  128.5338  128.5433    9.0001    0.0497    0.2118
starting iteration    9 best reduced chisq so far: 1.951E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   32.2864  0.510206  128.4739  128.5406    9.0001    0.0415    0.2419
s starting iteration   10 best reduced chisq so far: 1.932E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   32.1675  0.512825  128.5029  128.5426    9.0003    0.0362    0.2451
s starting iteration   11 best reduced chisq so far: 1.868E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   31.5550  0.541039  128.5865  128.5754    9.0007    0.0010    0.2587
starting iteration   12 best reduced chisq so far: 1.859E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   31.3262  0.544376  128.5704  128.5741    9.0001   -0.0128    0.2681
starting iteration   13 best reduced chisq so far: 1.859E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   31.3064  0.544489  128.5576  128.5716    9.0000   -0.0136    0.2687

 Done minimization
 Total number of iterations:           13
 Minimum chi^2 found:    18.5869942    



 Bootstrap number:            4
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   45.0000  0.450000  130.5000  126.5000   90.0000    0.0000    0.0000
starting iteration    1 best reduced chisq so far: 2.311E+02
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   45.0000  0.450000  130.5000  126.5000   90.0000    0.0000    0.0000
starting iteration    2 best reduced chisq so far: 3.627E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   38.4922  0.481794  126.8671  127.6624   92.0004    0.0519    0.0389
starting iteration    3 best reduced chisq so far: 2.274E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   38.1641  0.461259  128.3063  128.3826   35.5274    0.0571    0.0535
s starting iteration    4 best reduced chisq so far: 2.008E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   36.8813  0.458166  128.6624  128.5063   19.6749    0.0440    0.0828
starting iteration    5 best reduced chisq so far: 1.940E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   36.5643  0.462279  128.6353  128.4851   16.3965    0.0351    0.0929
starting iteration    6 best reduced chisq so far: 1.897E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   36.3231  0.465453  128.6208  128.4832   12.0497    0.0280    0.0996
starting iteration    7 best reduced chisq so far: 1.866E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   36.1585  0.468215  128.5827  128.4713    9.0030    0.0226    0.1038
starting iteration    8 best reduced chisq so far: 1.850E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   36.1451  0.470988  128.4916  128.4359    9.0000    0.0202    0.1039
s starting iteration    9 best reduced chisq so far: 1.849E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   36.1527  0.471973  128.4578  128.4234    9.0000    0.0197    0.1029
starting iteration   10 best reduced chisq so far: 1.849E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   36.1535  0.472184  128.4546  128.4223    9.0002    0.0197    0.1025

 Done minimization
 Total number of iterations:           10
 Minimum chi^2 found:    18.4868813    



 Bootstrap number:            5
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   45.0000  0.450000  130.5000  126.5000   90.0000    0.0000    0.0000
starting iteration    1 best reduced chisq so far: 2.318E+02
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   45.0000  0.450000  130.5000  126.5000   90.0000    0.0000    0.0000
starting iteration    2 best reduced chisq so far: 3.716E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   38.8862  0.506050  126.5750  127.9224   98.4732   -0.0771    0.1145
starting iteration    3 best reduced chisq so far: 2.252E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   37.0809  0.518722  128.2551  128.8134   72.3497   -0.0535    0.1239
starting iteration    4 best reduced chisq so far: 2.134E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   36.6309  0.515973  128.2643  128.7543   68.1180   -0.0652    0.1407
starting iteration    5 best reduced chisq so far: 2.023E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   35.4490  0.524642  128.1799  128.7151   54.5362   -0.0862    0.1766
starting iteration    6 best reduced chisq so far: 2.003E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   35.3352  0.521800  128.1623  128.7287   52.6895   -0.0782    0.1746
starting iteration    7 best reduced chisq so far: 1.976E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   34.9977  0.512957  128.0964  128.7062   43.3639   -0.0521    0.1665
starting iteration    8 best reduced chisq so far: 1.976E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   34.9772  0.512192  128.1082  128.7123   41.8399   -0.0503    0.1663
starting iteration    9 best reduced chisq so far: 1.976E+01
   diskpa:   diskel:     xcen:     ycen:    rwarp:    wepsm:    wphim:
   34.9717  0.511827  128.1133  128.7144   41.5033   -0.0502    0.1663

 Done minimization
 Total number of iterations:            9
 Minimum chi^2 found:    19.7570114    



 Finished           5  bootstrap iterations
 Wrote out: EXAMPLE/VELS/FITS/OUT/warp.out
