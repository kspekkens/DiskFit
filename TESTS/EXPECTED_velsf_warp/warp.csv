disk pa,disk el,xcen,ycen,R_{warp},wepsm,wphim
  3.2100E+01,  4.9916E-01,  1.2853E+02,  1.2870E+02,  6.2257E+01,  3.0895E+00,  1.4809E+01
  3.5831E+01,  4.9758E-01,  1.2864E+02,  1.2830E+02,  6.8448E+01,  3.1480E+00,  7.2574E+00
  3.1306E+01,  5.4449E-01,  1.2856E+02,  1.2857E+02,  9.0000E+00,  3.1280E+00,  1.5391E+01
  3.6153E+01,  4.7225E-01,  1.2845E+02,  1.2842E+02,  9.0002E+00,  3.1612E+00,  5.8687E+00
  3.4966E+01,  5.1187E-01,  1.2812E+02,  1.2872E+02,  4.1373E+01,  3.0912E+00,  9.5299E+00
