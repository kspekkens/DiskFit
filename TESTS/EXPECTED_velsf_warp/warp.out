Minimization output, vels
-------------------------
Input files:
EXAMPLE/VELS/FITS/velsf_warp.inp
EXAMPLE/VELS/FITS/vels.fits   pixscale  1.00 arcsec/pix


FITS region:       x-low:      1   y-low:      1 x-range:    255 y-range:    255
FITS sampling:    regrad:  12.00   regpa:  45.00  regeps:   0.45 istpout:      6

Output model and (data-model) residuals files:
EXAMPLE/VELS/FITS/OUT/warp.mod
EXAMPLE/VELS/FITS/OUT/warp.res

Disk toggles:         PA: T     eps: T  center: T non-axi: F    phib: F
vels toggles:    interp0: T  radial: F    Vsys: T    warp: T      rw: T    welm: T   wphim: T

Input values
------------
disk PA, phi_d^prime (deg):           45.00
disk eps:                              0.45
x,y center (data units):             130.50  126.50


Vsys (km/s):                         110.00
Delta_ISM (km/s):                      1.00

r_w (data units):                     90.00
Warp eps welm:                         0.00
Warp PA wphim:                         0.00

No seeing correction applied
No model component smoothing applied
Uncertainties estimated via bootstrap:  seed:      -50  nunc:        5  junc:   -1.00



Best fitting values
-------------------
disk PA, phi_d^prime (deg):           32.93 +/-  2.35
disk eps:                              0.52 +/-  0.02
disk incl (deg):                      61.40 +/-  1.62
x,y center (data units):             128.50 +/-  0.12,  128.54 +/-  0.17

Vsys (km/s):                         100.01 +/-  0.24
r_w (data units):                      9.00 +/- 26.52
Warp eps welm:                        -2.39 +/-  1.74
Warp PA wphim:                        10.25 +/-  4.19






Minimization Details
---------------------
# points Dn used in fit:               1224
# iterations in minimization:            12
Minimum chi^2 found:              25.192669
Degrees of freedom in fit:             1191

Fitted velocity components                  (radii in data units, velocities in km/s):
       r        npts          Vt         eVt          Vr         eVr        Vm,t       eVm,t        Vm,r       eVm,r
--------------------------------------------------------------------------------------------------------------------
      3.00     54.32       62.83        5.47        0.00        0.00        0.00        0.00        0.00        0.00
      9.00    128.30      117.65        6.40        0.00        0.00        0.00        0.00        0.00        0.00
     15.00     65.86      131.71        6.24        0.00        0.00        0.00        0.00        0.00        0.00
     21.00     13.74      143.44        7.37        0.00        0.00        0.00        0.00        0.00        0.00
     27.00     13.05      144.80        1.59        0.00        0.00        0.00        0.00        0.00        0.00
     33.00     16.32      150.19        7.20        0.00        0.00        0.00        0.00        0.00        0.00
     39.00     20.45      152.62        3.75        0.00        0.00        0.00        0.00        0.00        0.00
     45.00     22.76      154.38        1.48        0.00        0.00        0.00        0.00        0.00        0.00
     51.00     25.65      155.18        0.39        0.00        0.00        0.00        0.00        0.00        0.00
     57.00     28.98      161.96        4.84        0.00        0.00        0.00        0.00        0.00        0.00
     63.00     33.62      163.84        6.01        0.00        0.00        0.00        0.00        0.00        0.00
     69.00     35.12      173.30        7.77        0.00        0.00        0.00        0.00        0.00        0.00
     75.00     38.84      178.32        6.87        0.00        0.00        0.00        0.00        0.00        0.00
     81.00     43.01      183.56        6.89        0.00        0.00        0.00        0.00        0.00        0.00
     87.00     46.98      190.04        4.59        0.00        0.00        0.00        0.00        0.00        0.00
     93.00     48.64      189.50        3.67        0.00        0.00        0.00        0.00        0.00        0.00
     99.00     53.20      191.30        0.81        0.00        0.00        0.00        0.00        0.00        0.00
    105.00     58.67      192.74        3.61        0.00        0.00        0.00        0.00        0.00        0.00
    111.00     60.81      192.23        3.71        0.00        0.00        0.00        0.00        0.00        0.00
    117.00     64.57      193.35        3.25        0.00        0.00        0.00        0.00        0.00        0.00
    123.00     68.83      192.74        2.09        0.00        0.00        0.00        0.00        0.00        0.00
    129.00     75.09      193.85        1.77        0.00        0.00        0.00        0.00        0.00        0.00
    135.00     77.34      192.88        2.15        0.00        0.00        0.00        0.00        0.00        0.00
    141.00     71.92      194.10        2.52        0.00        0.00        0.00        0.00        0.00        0.00
    147.00     53.46      201.23        3.25        0.00        0.00        0.00        0.00        0.00        0.00
